Source/DataTypes/EnumConverters.cs

```diff
@@ -1,8 +1,9 @@
-﻿﻿using System;
+﻿using System;
```

Comment:
Has the encoding of the source file changed ?
`editorconfig` not applied ?

Reply:
No Visual Studio adds automatically correct BOM headers for Utf8 text files.

Reply:
[EditorConfig](https://editorconfig.org/) is supported by many editors and IDEs. (Not just Visual Studio)😄

Reply:
> [EditorConfig](https://editorconfig.org/) is supported by many editors and IDEs. (Not just Visual Studio)😄

I know that 😄 it's just that VS adds by default BOM header to .cs files with Utf8 encoding.

Reply:
Restored the original header bytes.

Reply:
@H1Gdev We probably should have separate PR with clean-up of .cs BOM headers accoring to editor config:
```ini
[*.cs]
charset = utf-8-bom
```

Reply:
I don't think you need to do that.

If **we** follow `EditorConfig`, I think coding styles will be unified over time.
(If not all.)